_topic_maps/_topic_map.yml

```diff
@@ -3435,19 +3435,19 @@ Topics:
     Topics:
     - Name: Backing up applications on AWS STS using OADP
       File: oadp-aws-sts
-  - Name: OADP 1.2 Data Mover
+#  - Name: OADP 1.2 Data Mover
+#    Dir: installing
+#    Topics:
+#    - Name: Introduction to OADP Data Mover
+#      File: data-mover-intro
+#    - Name: Using Data Mover for CSI snapshots
+#      File: oadp-using-data-mover-for-csi-snapshots-doc
+#    - Name: Using OADP 1.2 Data Mover with Ceph storage
+#      File: oadp-12-data-mover-ceph-doc
+  - Name: OADP Data Mover
     Dir: installing
```

Comment:
this was meant to remark not  remove completely?

Reply:
Yes because we have a Data Mover section for 1.3
So we are safe to just cut it